### ISSUE TITLE
user_falls.csv - is this data required? Morrissey falls barrier status error

### ISSUE THREAD

**Issue by smnorris** [smnorris/bcfishpass#198]
Most records are `barrier_ind=f`, there is not much point in maintaining this dataset if records included are not barriers.

However, one feature Morrissey Falls is 15m high, definitely a barrier. Maybe some review of these records would be valuable.
https://github.com/smnorris/bcfishpass/blob/main/data/user_falls.csv#L14
![](https://files.slack.com/files-pri/T01TP83RWQG-F0360LXN50B/kyle_img_2992.jpg)

**Comment by smnorris** [comment]
Any  user identified barriers should be going into `user_barriers_definite`, I don't see any reason for a separate user falls table. This file is probably a legacy of trying to add falls from WWD and other sources - but all the features listed from that load are `barrier_ind=f`

**Comment by smnorris** [comment]
@nickw-CWF Any thoughts on merging `user_falls.csv` table into `user_barriers_definite.csv`?

**Comment by nickw-CWF** [comment]
@smnorris I think that it might actually be more useful to move the canyons/waterfalls contained in ```user_barriers_definite.csv``` into ```user_falls.csv```.  That way the model logic of using observations to override gradient barriers could also be applied to ```user_falls.csv``` if ```barrier_ind=t``` - only for the WCT model though. Is there any reason these barriers can't be treated like gradient barriers for the WCT model? Or has that logic been removed from the model? For points like the Grave Canyon, those are valid observations upstream.

In terms of manual review of some of these tables, we are hiring a GIS technician in the coming weeks who will split time between the CABD and WCRPs so I can put the new technician on some of that.

**Comment by smnorris** [comment]
Agreed - I should be thinking of the falls as an additional data source augmenting FISS etc, rather than a user control table. Any features with barrier_ind='f' could probably be removed from that table unless they are needed as placeholders to indicate that a falls has been reviewed.

Access model queries are per-species access - so they can be tuned as required:

https://github.com/smnorris/bcfishpass/blob/main/scripts/model/sql/model_access_wct.sql#L32
https://github.com/smnorris/bcfishpass/blob/main/scripts/model/sql/model_access_ch_co_sk.sql#L192

**Comment by smnorris** [comment]
Besides re-organizing the data tables slightly it might also be valuable to split streams at *all* falls, not just those that get pulled out as minimal barriers. This would make reporting on features like Grave Canyon easier (important features, but not treated as barriers in the model because of observations upstream).

**Comment by nickw-CWF** [comment]
That all makes sense to me!
